src/components/SwapPage.tsx

```diff
@@ -295,9 +294,12 @@ const SwapPage = (props: Props): ReactElement => {
             )
           })}
         </div>
-        <Box mt={3}>
+        <Box mt={3} width="100%">
```

Comment:
Is it possible to accomplish this utilizing flexbox and not explicitly setting width to `100%`? I can imagine it can be a bit of a process with the migration to mUI as well.

Reply:
Right. We don't need to give `100%` if we will use flexbox. Let me update this accordingly removing `SCSS` file of `Swap` page.